cloud: add release notes 20230815 (#14526)
pingcap/docs: tidb-cloud/tidb-cloud-release-notes.md

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.

## August 15, 2023

**General changes**

- [Data Service (beta)](https://tidbcloud.com/console/data-service) supports pagination for `GET` requests to improve the development experience.

>     For an endpoint using the `GET` request method, you can enable **Pagination** and paginate results by specifying `page` and `page_size` as query parameters when calling the endpoint. For example, to get the second page with 10 items per page, you can use the following command:

Author: ```suggestion
    For an endpoint using the `GET` request method, you can paginate results by enabling **Batch Operation** in **Advance Properties** and specifying `page` and `page_size` as query parameters when calling the endpoint. For example, to get the second page with 10 items per page, you can use the following command:
```

>     For `GET` requests, you can paginate results by enabling **Batch Operation** in **Advance Properties** and specifying `page` and `page_size` as query parameters when calling the endpoint. For example, to get the second page with 10 items per page, you can use the following command:

Author: ```suggestion
    For `GET` requests, you can paginate results by enabling **Pagination** in **Advance Properties** and specifying `page` and `page_size` as query parameters when calling the endpoint. For example, to get the second page with 10 items per page, you can use the following command:
```

    ```bash
    curl --digest --user '<Public Key>:<Private Key>' \
      --request GET 'https://<region>.data.tidbcloud.com/api/v1beta/app/<App ID>/endpoint/<Endpoint Path>?page=2&page_size=10'
    ```

    Note that this feature is available only for `GET` requests where the last query is a `SELECT` statement.

    For more information, see [Call an endpoint](/tidb-cloud/data-service-manage-endpoint.md#call-an-endpoint).

> - [Data Service (beta)](https://tidbcloud.com/console/data-service) supports caching endpoint response of your `GET` requests within a specified time-to-live (TTL) period.

Reviewer: Remove "your" to be consistent with L15
```suggestion
- [Data Service (beta)](https://tidbcloud.com/console/data-service) supports caching endpoint response of `GET` requests for a specified time-to-live (TTL) period.
```

    This feature decreases database load and optimizes endpoint latency.

    For an endpoint using the `GET` request method, you can enable **Cache Response** and configure the TTL period for the cache in **Advance Properties**.

    For more information, see [Advanced properties](/tidb-cloud/data-service-manage-endpoint.md#advanced-properties).

> - Disable the load balancing improvement for TiDB Dedicated clusters that are hosted on AWS and created after August 15, 2023, including:

Author: ```suggestion
- Disable the load balancing improvement for [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters that are hosted on AWS and created after August 15, 2023, including:
```
